tests/tools/nnpackage_run/src/args.cc

```diff
@@ -92,6 +130,13 @@ void Args::Initialize(void)
          "{exec}-{nnpkg}-{backend}.csv will be generated.\n"
          "e.g. nnpackage_run-UNIT_Add_000-acl_cl.csv.\n"
          "{nnpkg} name may be changed to realpath if you use symbolic-link.")
+    ("shape_compile", po::value<std::string>()->default_value("[]"),
+         "set shape of specified tensor before compilation\n"
+         "e.g. '[0, [1, 2], 2, []]' to set 0th tensor to [1, 2] and 2nd tensor to [].\n")
+    ("shape_exec", po::value<std::string>()->default_value("[]"),
```

Comment:
@hyunsik-yoon What about `shape_prepare` and `shape_run`? Our API provides `nnfw_prepare()` and `nnfw_run()` and these values are applied before corresponding API call.

Reply:
Thanks. Fixed.